### ISSUE TITLE
Unsupported protocol(ffmpeg http)

### ISSUE THREAD

**Issue by Macadoshis** [rickeylohia/plugin.video.stalkervod#17]
Hi,

Using the plugin for the very first time.
Here is the error I'm getting when opening a channel (itv) after successfully being listed in after entering a category :

```
2024-06-09 01:40:56.508 T:1791     info <general>: VideoPlayer::OpenFile: plugin://plugin.video.stalkervod/?action=tv_play&cmd=ffmpeg+http%3A%2F%2Flocalhost%2Fch%2F438_
2024-06-09 01:40:56.509 T:2961     info <general>: Creating InputStream
2024-06-09 01:40:56.510 T:2961  warning <general>: CreateLoader - unsupported protocol(ffmpeg http) in ffmpeg http://hidden.domain.blablabla:8080/WErgHKPi2k/o7jVoBQ3Ii/438?play_token=hgoDiRHQ3u
2024-06-09 01:40:56.510 T:2961    error <general>: CVideoPlayer::OpenInputStream - error opening [plugin://plugin.video.stalkervod/?action=tv_play&cmd=ffmpeg+http%3A%2F%2Flocalhost%2Fch%2F438_]
2024-06-09 01:40:56.510 T:2961     info <general>: CVideoPlayer::OnExit()
2024-06-09 01:40:56.675 T:1791     info <general>: CVideoPlayer::CloseFile()
2024-06-09 01:40:56.676 T:1791     info <general>: VideoPlayer: waiting for threads to exit
2024-06-09 01:40:56.677 T:1791     info <general>: VideoPlayer: finished waiting
```

Did I miss something ?

Fyi: said channel opens successfully in same kodi instance, through the classical Stalker plugin (PVR).

KODI version : 20.2.0 (2023-06-29)
Plugin version : rickey - 0.0.6

**Comment by rickeylohia** [comment]
@Macadoshis 
issue should be fixed with v1.0.0. But I haven't pushed it to the official repo yet. Can you guys use below zip to install the latest version and test it out before I push it to the official repo.
If you guys can let me know the result either way if everything looks good / there are still any issues you guys see.

[plugin.video.stalkervod-1.0.0.zip](https://github.com/user-attachments/files/16120381/plugin.video.stalkervod-1.0.0.zip)

**Comment by rickeylohia** [comment]
issues should be fixed with v1.0.0 and be available in official kodi repo soon
